include/functions.h

```diff
@@ -2759,9 +2759,9 @@ void func_80143148(SkyboxContext* skyboxCtx, s32 arg1);
 void func_801431E8(GameState* gameState, SkyboxContext* skyboxCtx, s16 skyType);    
 void func_80143324(GlobalContext* globalCtx, SkyboxContext* skyboxCtx, s16 skyType);
 void func_801434E4(GameState* gameState, SkyboxContext* skyboxCtx, s16 skyType);    
-// void func_801435A0(void);
-// void func_80143624(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE1 param_5, UNK_TYPE1 param_6, UNK_TYPE1 param_7);
-// void func_80143668(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE4 param_6, UNK_TYPE4 param_7);
+// void SkyboxDraw_UpdateMatrix(void);
+// void SkyboxDraw_SetColors(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE1 param_5, UNK_TYPE1 param_6, UNK_TYPE1 param_7);
+// void SkyboxDraw_Draw(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE4 param_6, UNK_TYPE4 param_7);
```

Comment:
If you know what these functions are now, and they are used in other files (as I think is the case here?), you should fill in the correct prototypes for them. Otherwise, remove them.